### PR TITLE
Add Kubernetes Operator 0.1.0 release

### PR DESCRIPTION
**Pull request by gyfora**

### CODE REVIEW

**Comment by gyfora** [comment]
cc @mbalassi

**Comment by mbalassi** [comment]
Working with @gyfora on this, will merge when we are ready with the release.

File: _posts/2022-04-01-release-kubernetes-operator-0.1.0.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+The [official 0.1.0 release archive](https://archive.apache.org/dist/flink/flink-kubernetes-operator-0.1.0) doubles as a Helm repository that you can easily register locally:
+
+{% highlight bash%}
+$ helm repo add flink-kubernetes-operator-0.1.0 https://archive.apache.org/dist/\
+flink/flink-kubernetes-operator-0.1.0
+$ helm install flink-operator flink-kubernetes-operator-0.1.0/flink-operator --set webhook.create=false
+{% endhighlight%}
+
+You can also find official Kubernetes Operator Docker images of the new version on [Dockerhub](https://hub.docker.com/r/apache/flink-kubernetes-opeartor).
```
Comment (jbusche) [inline _posts/2022-04-01-release-kubernetes-operator-0.1.0.md]: Typo here, I think it should be: https://hub.docker.com/r/apache/flink-kubernetes-operator?

Comment (gyfora) [reply _posts/2022-04-01-release-kubernetes-operator-0.1.0.md]: good catch,thank you

**Comment by gyfora** [comment]
@mbalassi please build, review and merge if all good :) 
I have modified the blogpost release date to 04/04 (monday)

**Comment by mbalassi** [comment]
@gyfora roger that, reviewing and merging asap.

**Pull request closed without merging**